.github/workflows/release.yml

```diff
@@ -54,6 +54,8 @@ jobs:
         python -m pip install --upgrade pip
         pip install -r requirements.txt
         pip install setuptools wheel
+        pip install tensorflow
```

Comment:
Nit: May be combine into one line? Your call.

Reply:
I think it's fine